enshrined/svg-sanitize/CVE-2022-23638.yaml

```diff
@@ -0,0 +1,8 @@
+title:     A cross-site scripting vulnerability
+link:      https://github.com/darylldoyle/svg-sanitizer/releases/tag/0.15.0
+cve:       CVE-2022-23638
+branches:
+    master:
+        time:     2022-01-19 12:00:00
```

Comment:
```suggestion
        time:     2021-10-11 23:08:26
```

Reply:
Neither this nor the original time seem accurate? It looks like the fix was merged `2022-02-13 01:42:00`?

Reply:
refering to time of this merge commit: https://github.com/darylldoyle/svg-sanitizer/commit/17e12ba9c2881caa6b167d0fbea555c11207fbb0

Reply:
@naderman applied that time.